pkg/minishift/clusterup/clusterup.go

```diff
@@ -58,10 +60,12 @@ type ClusterUpConfig struct {
 	OcPath           string
 	AddonEnv         []string
 	PublicHostname   string
+	SSHCommander     provision.SSHCommander
+	OcBinaryInsideVM string
```

Comment:
Should be `OcBinaryPathInsideVM` or `OcPathInVM` but `Path` should be added to it.